src/infractl/deploy/gcp/main.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+"""
+Helper functions to deploy ICL cluster to GCP. These are called by scripts/deploy/gke.sh
+Uses Terraform to create a new GKE cluster and deploy infractl workloads
+"""
+
+import os
+import sys
+from subprocess import CalledProcessError, run
```

Comment:
See https://google.github.io/styleguide/pyguide.html#22-imports
> Use import statements for packages and modules only, not for individual types, classes, or functions.

```suggestion
import subprocess
import sys
```

Reply:
Resolved in commit:
https://github.com/intel-ai/icl/pull/76/commits/0de90d5ba55ef61cead0e0144be856e661c7e49d